package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "lint": "eslint **/*.ts",
     "build-all": "node build.js",
-    "build-diff": "node build.js --diff \"$(git diff --name-only)\"",
+    "build-diff": "node build.js --diff \"$(git diff --staged --name-only)\"",
```

Comment:
@LouisBoudreau you prefer to only build what is staged? I'm not sure if this could lead to some confusion 🤔

Reply:
J’ai ajouté ce flag car sans cela, nous nouvelles actions qui ne sont pas encore tracked ne sont pas build.
Donc en ajoutant une nouvelle action dans le projet, je faisais un commit et le build n’était pas trigger.
Donc avec le `--staged` on se retrouve à build les nouvelles actions aussi.  De plus, j’avais l’impression que l’on veut seulement build les actions qui ont été modifiées donc si elles devraient être  `--staged`